src/ontology/uberon.Makefile

```diff
@@ -1168,95 +1184,114 @@ composites: composite-metazoan.obo composite-vertebrate.obo
+# directly from the PURL source.
+$(TMPDIR)/uberon-cl.sssom.tsv: $(SRC) mirror/cl.owl $(TMPDIR)/plugins/sssom.jar
+	if [ -f mirror/cl.owl ]; then \
+		CL_INPUT='-i mirror/cl.owl' ; \
+	else \
+		CL_INPUT='-I http://purl.obolibrary.org/obo/cl/cl-base.owl' ; \
+	fi && \
+	$(ROBOT) merge -i $< $$CL_INPUT --collapse-import-closure false \
+		 sssom:xref-extract --mapping-file $@ -v --drop-duplicates \
+		                    --prefix 'DHBA:  http://purl.obolibrary.org/obo/DHBA_'  \
+		                    --prefix 'EFO:   http://purl.obolibrary.org/obo/EFO_'   \
+		                    --prefix 'HBA:   http://purl.obolibrary.org/obo/HBA_'   \
+		                    --prefix 'KUPO:  http://purl.obolibrary.org/obo/KUPO_'  \
+		                    --prefix 'OGES:  http://purl.obolibrary.org/obo/OGES_'  \
+		                    --prefix 'PBA:   http://purl.obolibrary.org/obo/PBA_'   \
+		                    --prefix 'SCTID: http://purl.obolibrary.org/obo/SCTID_' \
+	> $(REPORTDIR)/xrefs-extraction.txt
+
+# 2. Likewise, but from ZFA (ZFA is the source of truth for the CL-ZFA
+# mappings).
+$(TMPDIR)/zfa.sssom.tsv: mirror/zfa.owl $(TMPDIR)/plugins/sssom.jar
+	if [ -f mirror/zfa.owl ]; then \
+		ZFA_INPUT='-i mirror/zfa.owl' ; \
+	else \
+		ZFA_INPUT='-I http://purl.obolibrary.org/obo/zfa/zfa-base.owl' ; \
+	fi && \
+	$(ROBOT) merge $$ZFA_INPUT sssom:xref-extract --mapping-file $@
+
+# 3. Prepare the ruleset file.
+# The ruleset file is maintained with M4 macros to make it more easily
+# editable, so we need to expand the macros first.
+$(TMPDIR)/bridges.rules: $(SCRIPTSDIR)/sssomt.m4 $(BRIDGEDIR)/bridges.rules.m4
+	m4 $^ > $@
+
+# 3. Compile all mapping sets and generate the bridges.
+# Note that merging CL here is not strictly necessary, but doing so
+# allows sssom-inject to filter out any mapping with an inexistent or
+# obsolete Uberon/CL class.
+$(TMPDIR)/bridges: $(SRC) mirror/cl.owl \
+		   $(TMPDIR)/uberon-cl.sssom.tsv $(TMPDIR)/zfa.sssom.tsv $(EXTERNAL_SSSOM_SETS) \
+		   $(TMPDIR)/plugins/sssom.jar $(TMPDIR)/bridges.rules $(BRIDGEDIR)/bridges.dispatch \
+		   $(CUSTOM_BRIDGES)
+	if [ -f mirror/cl.owl ]; then \
+		CL_INPUT='-i mirror/cl.owl' ; \
+	else \
+		CL_INPUT='-I http://purl.obolibrary.org/obo/cl/cl-base.owl' ; \
+	fi && \
+	$(ROBOT) merge -i $(SRC) $$CL_INPUT \
+		 sssom:sssom-inject --sssom $(TMPDIR)/uberon-cl.sssom.tsv \
+		                    --sssom $(TMPDIR)/zfa.sssom.tsv \
+		                    $(foreach set, $(EXTERNAL_SSSOM_SETS), --sssom $(set)) \
+		                    --ruleset $(TMPDIR)/bridges.rules \
+		                    --dispatch-table $(BRIDGEDIR)/bridges.dispatch && \
+	touch $@
+
+# The above step creates RDF/XML bridges, turn them to OBO.
+# FIXME: Is this really necessary? Previously the OBO files were an
+# intermediate towards the final bridge files in OWL, but now we can
+# generate bridge files in OWL (or any format we want) directly, so
+# I don't think we need to carry over OBO files.
```

Comment:
I agree, but maybe double check with @cmungall whether the OBO files are needed as a release artefact. I don't think so!